canonical-web-and-design/ubuntu.com
# Replaced NextCloud with Nextcloud everywhere

## Done

- Replaced NextCloud with Nextcloud everywhere

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/appliance/nextcloud
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that is says 'Nextcloud' and NOT 'NextCloud'

## Issue / Card

Fixes #7953
